operator/pkg/model/model.go

```diff
@@ -114,13 +114,13 @@ func (l *TLSListener) GetPort() uint32 {
 type Service struct {
 	// Type is the type of service that is being used for Listener (e.g. Load Balancer or Node port)
 	// Defaults to Load Balancer type
-	Type string `json:"serviceType,omitempty"`
+	Type string `json:"type,omitempty"`
 	// InsecureNodePort is the back-end port of the service that is being used for HTTP Listener
 	// Applicable only if Type is Node NodePort
-	InsecureNodePort *uint32 `json:"insecureNodePort,omitempty"`
+	InsecureNodePort *uint32 `json:"insecure_node_port,omitempty"`
 	// SecureNodePort is the back-end port of the service that is being used for HTTPS Listener
 	// Applicable only if Type is Node NodePort
-	SecureNodePort *uint32 `json:"secureNodePort,omitempty"`
+	SecureNodePort *uint32 `json:"secure_node_port,omitempty"`
```

Comment:
I note that you rename some JSON fields in `model.go`, when the commit from `main` simply added new ones. Can this have impact elsewhere (I mean, do you need to update these field names in existing JSON files somewhere?)

Reply:
these tags are used for json serializing for test data only, it was originally planned, but then got removed in main.